gateways/core/resource/config.go

```diff
@@ -39,8 +39,6 @@ type resource struct {
 	Namespace string `json:"namespace"`
 	// Filter is applied on the metadata of the resource
 	Filter *ResourceFilter `json:"filter,omitempty"`
-	// Version of the source
-	Version string `json:"version"`
```

Comment:
This conflicts with `Version` of inline `GroupVersionKind`. I think we can remove it.